src/stlink-lib/chipid.c

```diff
@@ -8,6 +8,13 @@
 #include <ctype.h>
 #include <stdlib.h>
 
+// This is the old chipid "database". 
+// It is kept here for now to be able to compare the
+// result between the "old code" and the "new code". 
+// For now if you need to change something, please 
+// change it both here and in the corresponding 
+// config/chips/*.chip file. 
+
```

Comment:
I see the point, but am pretty sure this will not work out as intended for future contributions, as known from experience some people don't seem to be able to read or understand - I don't know. How ever we can leave this here for now until we are about to conclude this PR together. Then we should do a final comparison, test some MCU (which are available) and then sweep out the old stuff.

Reply:
Agreed.